### ISSUE TITLE
Model with CoordinateCouplerConstraint is not visualized correctly

### ISSUE THREAD

**Issue by tkuchida** [opensim-org/opensim-gui#857]
### Steps to reproduce
Open MI02_2392_patper_adjMaxForce.osim (from https://simtk.org/projects/crouchgait).
Attached here for convenience: [MI02_2392_patper_adjMaxForce.osim.txt](https://github.com/opensim-org/opensim-gui/files/2118163/MI02_2392_patper_adjMaxForce.osim.txt)

### Expected result
Model opens and is visualized as in 3.3:

![](https://user-images.githubusercontent.com/4203505/41641591-90662390-741a-11e8-8d0c-251f44849191.png)

### Actual result
Visualization is incorrect:

| Default pose | With knee flexed |
| --- | --- |
| ![](https://user-images.githubusercontent.com/4203505/41641603-9e4a79ac-741a-11e8-8ec1-26d3f2385a4b.png) | ![](https://user-images.githubusercontent.com/4203505/41641626-b772bb7e-741a-11e8-8a0f-789d24c00936.png) |

### Environment and GUI version
OpenSim-1e476f28-2018-06-13-win64.exe from Dropbox ("OpenSim 4.0 User Session and Tutorial" version); no local hacks.

**Comment by aymanhab** [comment]
I suspect this is not a visualization issue but rather an issue with interpretation of coordinates as rotations that have changed leading to via-points activating incorrectly and other downstream effects.

**Comment by aymanhab** [comment]
Thanks @tkuchida The model file has a Document version of 10600 which is before version 2.0. After loading the model in version 3.3 and saving it, the model loads and visualizes without a problem in 4.0. Need to decide the desired behavior/feedback to these old format model files.

**Comment by tkuchida** [comment]
> After loading the model in version 3.3 and saving it, the model loads and visualizes without a problem in 4.0.

Then is this a problem with `updateFromXMLNode()`?

**Comment by aymanhab** [comment]
This is possible. Basically the updateFromXML mechanism handles changes within a class rather smoothly but could fail when there are significant structural changes (e.g. classes removed, or split or XML layout change) We should iron out what versions are supported (I think those based on version 2.4 and later are) and give an error message suggesting that users save the model in version 3.3 instead of waiting for problems downstream.

**Comment by aymanhab** [comment]
I suggest we move this to opensim-core to detect file version and if not supported throws exception,  if GUI doesn't handle/display an informative message then we can revisit. This would keep the fine control of what's supported in the hands of opensim-core without exposing too much of the version/support internals.

**Comment by tkuchida** [comment]
> I suggest we move this to opensim-core

Fine with me.

**Comment by aseth1** [comment]
I don't believe this is a degrees vs. radians issues. There are several potential problems. First. the translation and rotational coordinates of the pelvis in this model are swapped. Attempting to visualize with the API throws an Exception that a NaN value was returned:
```
Model::equilibrateMuscles() Unable to compute equilibrium for this muscle.
Please verify that the initial activation is valid and that the length
simbody-visualizer: received Shutdown message. Goodbye.
 of the musculotendon actuator doesn't produce a pennation angle of 90 degrees or a negative fiber length.

  Solution error 7.74347e+07 exceeds tolerance of 5.37e-06
  Newton iterations reached limit of 20
  Activation is 0.05
  Fiber length is nan

        In Object 'rect_fem_r' of type Thelen2003Muscle.
        Thrown at Thelen2003Muscle.cpp:376 in computeInitialFiberEquilibrium().
```

It should have failed to load in the GUI due to NaNs. So I would leave this issue open on the GUI side as well.
